content/analyst/competitive-tracker/sending-domain.md

```diff
@@ -0,0 +1,72 @@
+---
+lastUpdated: "02/23/2023"
+title: Sending Domain
+description: What insights can I learn from the Sending Domain page of Competitive Tracker?
+---
+
+[**Here is a video going over all the data on the Sending Domain page:**](https://www.youtube.com/embed/pwuBpFFKRfM)
```

Comment:
This one is linked correctly but is written as if the video was embedded (was on intercom)